docs/quickstarts/twitter.rst

```diff
@@ -54,9 +54,49 @@ run:
     $ export OAUTHLIB_INSECURE_TRANSPORT=1
     $ python twitter.py
 
-Visit http://localhost:5000 in your browser, and you should start the OAuth dance
+Visit http://127.0.0.1:5000 in your browser, and you should start the OAuth dance
 immediately.
 
+In your view, make sure you use a plain hyperlink to begin the dance so that the redirect loads Twitter's confirmation page in the browser. 
+
+```
```

Comment:
This is not how you do code blocks in rst files. They start with `.. code-block:: python` or whichever language as you can see in the rest of the file. The code then needs to be indented by 4 spaces.

Reply:
@daenney ahhh whoops sorry about that - obviously  I missed the rst format - so I've now research the format, added an rst plugin and linter to my vs code and have fixed those issues. 

thanks for the review!